Magitek/Views/UserControls/Sage/Shield.xaml.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Windows.Controls;
+
+namespace Magitek.Views.UserControls.WhiteMage
```

Comment:
These whitemage / regen references need to change to sage / shield, then this will build and we can merge.